migrating/ossm-migrating-read-me-assembly.adoc

```diff
@@ -15,8 +15,6 @@ include::modules/ossm-migrating-read-me-new-operator.adoc[leveloffset=+1]
 You can install the {SMProduct} 3 Operator, or you can run {SMProduct} 2.6 and {SMProduct} 3 in the same cluster using either the multi-tenant deployment model, or the cluster-wide model.
```

Comment:
Please also remove Next steps chapter. We don't want to instruct users to install 2.6 and 3.0 together.